wings/market/ddex_market.pyx

```diff
@@ -818,15 +845,47 @@ cdef class DDEXMarket(MarketBase):
 
         return order_book.c_get_price(is_buy)
 
-    cdef c_start(self, Clock clock, double timestamp):
-        MarketBase.c_start(self, clock, timestamp)
+    async def start_network(self):
+        await super().start_network()
```

Comment:
Why is it calling `super().start_network` -- it seems like MarketBase does not define it and the NetworkIterator class only defines it in abstract